### PR TITLE
Add support for OL7

### PR DESCRIPTION
**Pull request by gstankovski**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Configures yum repos on an Oracle Linux system'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '1.0.1'
+version '1.0.2'
```
Comment (juliandunn) [inline metadata.rb:7]: Please don't modify metadata - I'll do that

**Comment by juliandunn** [comment]
Overall seems legit, but I'd like to not turn off GPG checking. Can you fix that?

**Comment by gstankovski** [comment]
Updated gpgcheck to default to true

File: metadata.rb (hunk 1 above)
Comment (gstankovski) [inline metadata.rb:7]: Ok, will do.

**Comment by juliandunn** [comment]
Thanks for your contribution. I squashed and merged it in 4c7dbb3b312b87c6a9abab4c5b30d40dec73c8e8!

**Pull request closed without merging**